tests/test_data.py

```diff
@@ -10,30 +10,32 @@ def create_altinn_test_catalog() -> Catalog:
 
     altinn_test_catalog.identifier = "https://www.altinn.no/models/catalog"
     altinn_test_catalog.title = {"nb": "Altinn informasjonsmodellkatalog"}
-    altinn_test_catalog.publisher = "https://organization-catalogue.fellesdatakatalog.digdir.no/organizations/991825827"
+    altinn_test_catalog.publisher = "https://organization-catalogue.staging.fellesdatakatalog.digdir.no/organizations/991825827"
 
```

Comment:
Ser det er endringer i formattering. Til en annen gang kan det være fint å ha slike ting i en egen chore, rett og slett bare for lesningheten av PR'en sin skyld. :-)